media/css/new-start-style.css

```diff
@@ -0,0 +1,299 @@
+  font-family: system-ui;
+  font-style: normal;
+  font-weight: 400;
+  src: local(".SFNS-Regular"), local(".SFNSText-Regular"), local("Segoe UI"), local("Ubuntu");
+}
+
+@font-face {
+  font-family: system-ui;
+  font-style: italic;
+  font-weight: 400;
+  src: local(".SFNS-Italic"), local(".SFNSText-Italic"), local("Segoe UI Italic"), local("Ubuntu Italic");
+}
+
+@font-face {
+  font-family: system-ui;
+  font-style: normal;
+  font-weight: 500;
+  src: local(".SFNS-Medium"), local(".SFNSText-Medium"), local("Segoe UI Semibold"), local("Ubuntu Medium");
+}
+
+@font-face {
+  font-family: system-ui;
+  font-style: italic;
+  font-weight: 500;
+  src: local(".SFNS-MediumItalic"), local(".SFNSText-MediumItalic"), local("Segoe UI Semibold Italic"), local("Ubuntu Medium Italic");
+}
+
+@font-face {
+  font-family: system-ui;
+  font-style: normal;
+  font-weight: 700;
+  src: local(".SFNS-Bold"), local(".SFNSText-Bold"), local("Segoe UI Bold"), local("Ubuntu Bold");
+}
+
+@font-face {
+  font-family: system-ui;
+  font-style: italic;
+  font-weight: 700;
+  src: local(".SFNS-BoldItalic"), local(".SFNSText-BoldItalic"), local("Segoe UI Bold Italic"), local("Ubuntu Bold Italic");
+}
+
+:root {
+  --bg: #f9f9fa;
+  --bg-gradient: linear-gradient(-45deg, #F9F9FA, #EDEDF0);
+  --bg-warning: rgba(237, 0, 181, 0.05);
+
+  --beta-gradient: linear-gradient(-270deg, #E217B6, #BD058A);
+  --logo-wireframe: url('/media/svg/logo-wireframe-light.svg');
+
+  --text: #333;
+  --text-logo: #0a84ff;
+  --text-donate: #ff0039;
+  --text-connect: #9400ff;
+  --text-support: #0a84ff;
+  --text-get-involved: #ed00b5;
+  --text-bug: #058B00;
+  --text-crash: #008EA4;
+  --text-warning: #B5007F;
+
+  --shadow: 0 4px 16px rgba(0, 0, 0, 0.1);
+}
+@media (prefers-color-scheme: dark) {
+  :root {
+    --bg: #2F2F33;
+    --bg-gradient: linear-gradient(-45deg, #2F2F33, #3A3A3D);
+    --bg-warning: rgba(181, 0, 127, 0.05);
+
+    --beta-gradient: linear-gradient(-270deg, #FF83EA, #FF1AD9);
+    --logo-wireframe: url('/media/svg/logo-wireframe-dark.svg');
+
+    --text: #F9F9FA;
+    --text-logo: #85C2FF;
+    --text-donate: #F99298;
+    --text-connect: #D9A8FF;
+    --text-support: #85C2FF;
+    --text-get-involved: #FFB6F3;
+    --text-bug: #ABF69B;
+    --text-crash: #B0FFFF;
+    --text-warning: #FFB6F3;
+
+    --shadow: 0 4px 16px rgba(0, 0, 0, 0.3);
+  }
+}
+
+html {
+  font-family: system-ui, Open Sans, sans-serif;
+  background-color: var(--bg);
+  background-image: var(--bg-gradient),
+}
+
+body {
+  background-image: var(--logo-wireframe);
+  background-position: bottom right;
+  background-repeat: no-repeat;
+  color: var(--text);
+  margin: 0;
+  padding: 0;
+  min-height: 100vh;
+  max-width: 100%;
+  line-height: 1.6;
+}
+
+main {
+  display: grid;
+  gap: 2rem;
+  padding: 1rem;
+  margin: 0 auto;
+  max-width: 100ch;
+}
+
+header {
+  display: grid;
+  gap: 2rem;
+}
+
+svg {
+  fill: currentColor;
+  vertical-align: -4px;
+  margin-right: 0.25rem;
```

Comment:
Better to avoid these declarations for a global selectors. With this, all SVGs in the page will have this alignment and margins, whether we need it or not.
Better create a specific class for icons in titles and style that.